src/v/cloud_storage/remote.h

```diff
@@ -207,6 +207,17 @@ class remote : public ss::peering_sharded_service<remote> {
       const remote_segment_path& path,
       retry_chain_node& parent);
 
+    /// \brief Delete segment from S3
+    ///
+    /// The method deletes the segment. It can retry after some errors.
+    ///
+    /// \param segment_path is a segment's name in S3
+    /// \param manifest is a manifest that should have the segment metadata
```

Comment:
does this need to be removed, as there is no manifest?

Reply:
yep

Reply:
fixed